### PR TITLE
DP-731 - Duplicate Checks - Additional Identifier and direct user to request to join

### PR DESCRIPTION
**Pull request by rmohammed-goaco**
https://noticingsystem.atlassian.net/browse/DP-731

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml [hunk 1]
```diff
@@ -32,10 +32,9 @@
                     If you know the person who registered the organisation, they can invite you to join. Or you can request to join below.
                 </p>
 
-                <govuk-button>
+                <a class="govuk-button" href="/registration/@Model.Identifier/join-organisation">
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml:35]: If we're no longer posting a form with a button here, then the form on line 22 should go as well

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml.cs [hunk 2]
```diff
@@ -1,7 +1,16 @@
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
 namespace CO.CDP.OrganisationApp.Pages.Registration;
 
 public class OrganisationAlreadyRegistered: PageModel
 {
+    [BindProperty]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml.cs:8]: Bit confused how this page is being used? `[BindProperty]` is for data that is coming from get or post data. The identifier is already sufficiently retrived from the URL segment (`@page "/registration/organisation-already-registered/{Identifier}"`) by the `OnGet(string identifier)` so you don't need the Bind Property at all?

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml.cs [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+    public string? Identifier { get; set; }
+
+    public IActionResult OnGet(string identifier)
+    {
+        Identifier = identifier;
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml.cs:13]: Bit confused how this page is being used? `[BindProperty]` is for data that is coming from get or post data. The identifier is already sufficiently retrived from the URL segment (`@page "/registration/organisation-already-registered/{Identifier}"`) by the `OnGet(string identifier)` so you don't need the Bind Property at all?

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml.cs (hunk 3 above)
Comment (rmohammed-goaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml.cs:13]: I'll try without and see what happens

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml (hunk 1 above)
Comment (rmohammed-goaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml:35]: Will remove

**Review by andymantell — APPROVED** [review]

**Review by dharmverma — APPROVED** [review]

**Pull request merged**